docs/pbc/all/data-exchange/202311.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md

```diff
@@ -302,7 +302,7 @@ For example, configure `isCreatable: false` for `iso3_code` and send the same re
 ```json
 [
   {
-    "message": "Modification of immutable field `iso3_code` is prohibited.",
+    "message": "Modification of immutable field `countries[0].iso3_code` is prohibited.",
```

Comment:
In reality, the chain looks differently (but must look like it is done in documentation), https://github.com/spryker/robotframework-suite-tests/pull/865 please create a bug and fix it.

Reply:
Do you mean putting the index in the brackets? If yes, in reality, it works the same as in documentation. But I saw this "weirdo" behavior in robot framework and I do not know how to explain that

Reply:
Ok, thanks